### ISSUE TITLE
Error immediately on selecting "Custom Character"

### ISSUE THREAD

**Issue by MarcusVoelker** [TGWeaver/CDDA-Sky-Islands#16]
As the title says, when trying to create a Custom Character, the error below occurs and the game returns to the main menu.

The game is a fresh install of F-3 via catapult

```
 DEBUG    : Error: Json error: data/mods/Sky_Islands/EOCs.json:3:12: unrecognized JSON object

[
  {
    "type":
           ^
            "effect_on_condition",
    "id": "EOC_difficultysafeguard",
    "//": "This EOC gets called after a teleport is initialized but before it is finished. It basically just checks if you're running an older save or somehow missed the difficulty


 FUNCTION : bool main_menu::new_character_tab()
 FILE     : src/main_menu.cpp
 LINE     : 865
 VERSION  : 0.F-3
```

Contrary to what the game claims, there is no full stack trace at the path it gives me, so this is all I have

**Comment by gitx-xx** [comment]
Seems like a cool mod but I get the same message.
I'm running Arch, and installed Cataclysm via AUR (terminal version), and git cloned this repo into ~/.local/share/cataclysm-dda/mods/CDDA-Sky-Islands/.
The mod seems to load fine in cataclysm and world generation, but immediately after picking 'Custom Character' the error message above shows. None of the alternatives in character creation is working.
Ideas?

**Comment by amcmullan** [comment]
I got this too, windows 10 64-bit. 

Installed and ran via cdda launcher with mod copied in. Just the basic includes:
dark days ahead
disable NPC needs
bionic professions
sky island

**Comment by TGWeaver** [comment]
The problem is right here:
`VERSION  : 0.F-3`
You're using the stable release version, which is over a year old. This mod will only run on the new experimental version. Any release from within the past couple weeks should probably run fine.

**Comment by gitx-xx** [comment]
Compiling latest source of cataclysm solved this issue for me. Thanks!

**Comment by MarcusVoelker** [comment]
Same here. Might I suggest updating the README to reflect this?

**Comment by TGWeaver** [comment]
Good suggestion. Fixed in 4564855
